app/assets/javascripts/discourse/app/templates/user-private-messages-tags-nav.hbs

```diff
@@ -0,0 +1,14 @@
+<li class="tags">
+  <LinkTo @route="userPrivateMessages.tags.index" @model={{this.user.model}}>
+    {{d-icon "tag"}}
+    <span>{{i18n "user.messages.all_tags"}}</span>
+  </LinkTo>
+</li>
+
```

Comment:
<img width="1137" alt="Screenshot 2022-11-28 at 5 36 10 pm" src="https://user-images.githubusercontent.com/72780/204209976-5dbe738d-24ee-4614-8369-b1d2a5d8fc85.png">

That button is quite big. Also are we missing padding from grey line?

Reply:
Let us avoid focusing on the design changes for this change here as the changes here does not alter the original design. I'll address the changes outside of this PR in consultation with @awesomerobot